pkgs/by-name/go/goshs/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule rec {
+  pname = "goshs";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "patrickhener";
+    repo = "goshs";
+    tag = "v${version}";
+    hash = "sha256-J74nUH3iBYDytXCswIexvShG5+zJEDX19saJ3ftLYhI=";
+  };
+
+  vendorHash = "sha256-6UaKh2UUyGqOriAaMhiEFr20R4W/ZWxQXtXnm/BVHOc=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  meta = {
+    description = "Simple, yet feature-rich web server written in Go";
+    homepage = "https://goshs.de";
+    license = lib.licenses.mit;
+    platforms = lib.platforms.all;
```

Comment:
I seriously doubt this (I don't think Go compiles for all platforms Nixpkgs supports). If this is supposed to work for all platforms Go supports, then just leave it blank